test/cli/suites-consumer/sdk.go

```diff
@@ -30,7 +30,7 @@ var _ = ginkgo.Describe("adc APISIX consumer SDK tests", func() {
 			baseConsumer1 := &types.Consumer{
 				Username: "consumer1",
 			}
-			_, err = s.CreateConsumer(baseConsumer1)
```

Comment:
this error should be checked

Reply:
I have used gomega to assert if errors are nil and entailed an optionalDescription if they are not matching. can you please review the error checks?